src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -284,6 +284,27 @@ public void waitForConfirmationModalAndClickOk() {
         click(okayButton);
         waitForModalToDisappear();
     }
+    
+    /**
+     * Waits for a confirmation modal to appear and click the No button
+     */
+    public void clickNoOnModal() {
+        waitForModalPresence();
+        WebElement noButton = browser.driver.findElement(By.cssSelector("[data-bb-handler='no']"));
+        waitForElementToBeClickable(noButton);
+        click(noButton);
+        waitForModalToDisappear();
+    }
+    
+    /**
+     * Waits for a confirmation modal to appear and click the confirm button, but doesn't wait for modal to disappear
+     */
+    public void waitForConfirmationModalAndClickOkWithoutWaitingForModalDisappearance() {
```

Comment:
This method is only called once in `clickAndConfirmWithoutWaitingForModalDisappearance`, right? Then there is no need to separate it; just move everything to `clickAndConfirmWithoutWaitingForModalDisappearance`.